tests/cargo-add.rs

```diff
@@ -1304,7 +1304,7 @@ fn adds_dependency_with_target_cfg() {
 }
 
 #[test]
-fn adds_features_dependency() {
+fn lists_features_github_dependency() {
```

Comment:
How come this test got renamed.  It looks like its still focused on validating adding a feature

Reply:
mistake on my part, I think I renamed the wrong test. I've corrected it.